site/en/docs/extensions/reference/commands/index.md

```diff
@@ -110,3 +107,133 @@ Example:
   ...
 }
 ```
+
+## Examples
+
+The following examples show
+
+### Basic command
+
+Commands allow extensions to map logic to keyboard shortcuts that can be invoked by the user. At
+it's most basic, a command only requires a command declaration in the extension's manifest and a
```

Comment:
its

Reply:
Some day I'll internalize this rule. Today is not that day.